src/garage/np/algos/base.py

```diff
@@ -6,12 +6,27 @@ class RLAlgorithm(abc.ABC):
 
     @abc.abstractmethod
     def train_once(self, itr, paths):
```

Comment:
We should probably remove this abstract method from the interface, since `train()` is the new entrypoint.

That can wait for a later PR though.

Reply:
In fact, please *don't* change that in this PR. It's ready to go, so let's not slow it down by adding *even more* changes.